test/functionalTest/cases/1164_v2_update_entities_attr_empty_value/update_entities_attr_empty_value.test

```diff
@@ -30,9 +30,11 @@ brokerStart CB
 --SHELL--
 
 #
-## 01. POST /v2/entities to create E1/A1=0
-## 02. PUT /v2/entities/E1/attrs/A1/value (empty)
-
+# 01. POST /v2/entities to create E1/A1=0
+# 02. GET /v2/entities/E1/attrs/A1
+# 03. PUT /v2/entities/E1/attrs/A1/value (empty)
```

Comment:
True, I didn't look very closely at the functests, just made them work again ...
But, I did remove the extra #s, and I will fix this also, of course ...